### PR TITLE
New package: Xiph.Flac version 1.4.3

### LINKED ISSUE #151382: [Package Request]: flac
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://xiph.org/flac/download.html
Publisher: Xiph
Package Name: flac
Description: Free Lossless Audio Codec
Package Version: 1.4.3
Installer URL: 
https://github.com/xiph/flac/releases
or
https://ftp.osuosl.org/pub/xiph/releases/flac/

### PR DESCRIPTION
**Pull request by schembriaiden**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Resolves: https://github.com/microsoft/winget-pkgs/issues/151382

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151387)

### CODE REVIEW

File: manifests/x/Xiph/Flac/1.4.3/Xiph.Flac.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Installers:
+- InstallerUrl: https://github.com/xiph/flac/releases/download/1.4.3/flac-1.4.3-win.zip
+  Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: flac-1.4.3-win\Win32\flac.exe
+  InstallerSha256: C4558CF79FC1365D1822F7945A20BDBF55F99642CEE95A823D44F3A61FB748C6
+- InstallerUrl: https://github.com/xiph/flac/releases/download/1.4.3/flac-1.4.3-win.zip
+  Architecture: x64
+  NestedInstallerFiles:
+  - RelativeFilePath: flac-1.4.3-win\Win64\flac.exe
```
Comment (sidy3d) [inline manifests/x/Xiph/Flac/1.4.3/Xiph.Flac.installer.yaml:17]: Should set `PortableCommandAlias: flac`

File: manifests/x/Xiph/Flac/1.4.3/Xiph.Flac.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: Xiph.Flac
+PackageVersion: 1.4.3
+InstallerType: zip
+NestedInstallerType: portable
+Installers:
+- InstallerUrl: https://github.com/xiph/flac/releases/download/1.4.3/flac-1.4.3-win.zip
+  Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: flac-1.4.3-win\Win32\flac.exe
```
Comment (sidy3d) [inline manifests/x/Xiph/Flac/1.4.3/Xiph.Flac.installer.yaml:12]: Should set `PortableCommandAlias: flac`

**Review by sidy3d — CHANGES_REQUESTED** [review]
PortableCommandAlias should be set to `flac` for both architectures.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\flac.exe returned exit code: -1073741515
> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\flac.exe returned exit code: -1073741515

(Automated response - build 877.)

**Comment by sidy3d** [comment]
The executables require `libFLAC.dll` and `libFLAC++.dll` which are included in the archive package. I am not sure how to reference them the manifest.

**Pull request closed without merging**